data-raw/simulate_longitudinal_data.R

```diff
@@ -1,3 +1,7 @@
+devtools::install('../gsm.mapping')
+devtools::install('../gsm.core')
+devtools::install('../gsm.kri')
+
```

Comment:
It's messy to assume everyone has the same directory structure. Install the needed versions from github (I'm not sure if these are exactly what you're aiming at because I'm not sure what was in your folders when you ran this):

```suggestion
pak::pak('Gilead-BioStats/gsm.mapping@dev')
pak::pak('Gilead-BioStats/gsm.core@dev')
pak::pak('Gilead-BioStats/gsm.kri@dev')

```